scripts/export_inspector_findings.py

```diff
@@ -198,7 +210,13 @@ def write_to_csv(self, findings: dict[str, list[SummaryType]]) -> None:
             row_num = len(rows) + 1
             col_range = f'C{row_num}:{last_col}{row_num}'
             severity_formula = f'=(COUNTIF({col_range},"C")*10)+(COUNTIF({col_range},"H"))'
-            row = [vulnerability, severity_formula]
+            urls = Counter([summary['source_url'] for summary in summaries])
+            if len(urls.keys()) > 1:
+                log.warning('More than one URL found for %s, using most common', vulnerability)
+                log.warning(dict(urls.most_common()))
+            url = urls.most_common(1)[0][0]
```

Comment:
```suggestion
            url, frequency = one(urls.most_common(1))  # REVIEW: my version is stricter and more readable
            if len(urls) > 1:
                log.debug('URLs by by frequency: %r', urls.most_common())
                log.warning('More than one URL found for %s, using the most common one (%r)', vulnerability, url)
 ```
 
Are you sure each URL can be listed more than once? If so, please explain your reasoning behind this heuristic. If not, please simplify the above.